alacritty/src/cli.rs

```diff
@@ -251,6 +253,14 @@ impl Options {
             wd => config.working_directory = wd,
         }
 
+        match &config.random_colors {
+            Some(colors) => {
+                info!("Available colors {:?}", colors);
+                config.colors = colors[rand::random::<usize>() % colors.len()].clone();
+            },
+            None => info!("No random colors found"),
+        }
+
```

Comment:
Apologies if this is not the place to do this.


I needed a mutable `config` and this looked like a good spot. Happy to move it if not appropriate.